tests/main/hidden-snap-dir/task.yaml

```diff
@@ -184,3 +184,22 @@ execute: |
     check_env "--with-exposed-home" "x3"
     check_dirs "--with-exposed-home" "x3"
     check_data "--with-exposed-home" "x3" "$data"
+
+    echo "Check migration after remove works"
+    # ensure dirs under ~/.snap/data are created
+    "$NAME".cmd 'true'
```

Comment:
Just to be sure, add a `test -e "$HOME/.snap/data/$NAME"` after this line.

Reply:
+1